.buildkite/publish.sh

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bash
+
+set -euv
+
+# The build agents have gpg 2.0.22, which doesn't have the `--pinentry-mode` option, but
+# the sbt-pgp plugin assumes that 2.x has it, and generates an invalid command.
+# We can either create a wrapper script that removes that option, or update gpg
+# somewhere in pipeline.nix
+
+# Force a restart of the agent becuase it may be out of sync with what Nix installed.
+gpgconf --kill all
+
+gpg --version
+
+
+# The build agent might have this key from before.
+GPG_EXISTS=$(gpg --list-keys "$GPG_KEY_ID" >&2 && echo "yes" || echo "no")
+
+if [[ "$GPG_EXISTS" == "no" ]]; then
+	echo "$GPG_KEY" | base64 --decode | gpg --batch --import
+fi
+# Local testing showed that without this the SBT plugin got "Bad passphrase".
```

Comment:
It turns out it's because without this the `--pinentry-mode loopback` doesn't work. When I removed that from the command the plugin printed, and tried it in the console, it worked with the `--passphrase` flag, so it's not an issue of surrounding with `'` or having a bad password.